packages/basics/package.json

```diff
@@ -19,7 +19,6 @@
   },
   "homepage": "https://github.com/Inist-CNRS/ezs/tree/master/packages/basics#readme",
   "dependencies": {
-    "JSONStream": "~1.3.5",
```

Comment:
Why remove it from dependencies ?
If default option is to use legacy parser...

Reply:
you're right thanks